packages/config/src/handlers/application/applicationConfigHandler.ts

```diff
@@ -36,7 +37,16 @@ export default async function configHandler(
       systems,
       formDataset
     ] = await Promise.all([
-      getActiveCertificatesHandler(request, h),
+      getActiveCertificatesHandler(request, h).then((certs) =>
+        Promise.all(
+          certs.map(async (cert) => ({
+            ...cert,
+            svgCode: await getDocumentUrl(cert.svgCode, {
+              Authorization: request.headers.authorization
+            })
+          }))
+        )
+      ),
```

Comment:
Maybe this could be handled by the `getActiveCertificatesHandler` ?

Reply:
hm let's see if that handler is only being used from here then we can move the logic inside. Alright I'll take a look. Thanks Pyry!

Reply:
we have a gql query that uses this route & the the presigned url is generated in the type resolvers there so we can't do this inside the handler I'm afraid